app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -655,16 +655,13 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
       align-items: center;
       padding: 0 var(--su-1);
       width: 100%;
+      display: inline-block;
 
       &:hover {
         background: var(--body-bg);
         cursor: pointer;
       }
 
-      &:last-child {
-        margin-bottom: 0;
-      }
```

Comment:
I removed this unnecessary style because this style does not have `margin-bottom`.